src/middlewares/validator.js

```diff
@@ -0,0 +1,30 @@
+import { validationResult } from 'express-validator/check';
+import { matchedData } from 'express-validator/filter';
+
+/**
+ * Generic schema validator
+ *
+ * @param {Array} schema
+ * @param {number} status
+ *
+ * @returns {object} returns response message
+ */
+
+export default (schemas, status = 400) => {
+  const schemaTovalidate = schemas || [];
```

Comment:
Really good job. Please update to 
```suggestion
  const schemaToValidate = schemas || [];
```